pkg/graphql/src/resolver/features/features.json

```diff
@@ -5,10 +5,104 @@
     "CREATE_PROPOSAL_RELATIVE_MAJORITY": false,
     "CREATE_GENERAL_PROPOSAL": true,
     "CREATE_WITHDRAW_PROPOSAL": true,
-    "CREATE_SPENDING_PROPOSAL": false
+    "CREATE_SPENDING_PROPOSAL": false,
+    "ORGANIZATION_PAGE_FEATURES": {
+      "SHOW_SEARCH": true,
+      "SHOW_FILTERS": true,
+      "SHOW_SORT": true,
+      "SORT_OPTIONS": [
+        {
+          "name": "Member: High-Low",
+          "value": "memberLimit_DESC"
+        },
+        {
+          "name": "Member: Low-High",
+          "value": "memberLimit_ASC"
+        },
+        {
+          "name": "Created: Newest first",
+          "value": "createdAtBlock_DESC"
+        },
+        {
+          "name": "Created: Oldest first",
+          "value": "createdAtBlock_ASC"
+        }
+      ]
+    }
   },
-  "dev": {
+  "DEVELOPMENT": {
```

Comment:
1. Why were the enviroments renamed?
2. It is enough if it is specified once in global
3. It should be all on one level only with value true and false. 
No nesting, No text values

Reply:
for testing purposes, and according to the enum in the gamedao.graphql file